TASK/TaskFollowPointRoute.md

```diff
@@ -5,14 +5,22 @@ ns: TASK
 
 ```c
 // 0x595583281858626E 0xB837C816
-void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int unknown);
+void TASK_FOLLOW_POINT_ROUTE(Ped ped, float speed, int flag);
 ```
 
 ```
 MulleKD19: Makes the ped go on the created point route.
 ped: The ped to give the task to.
 speed: The speed to move at in m/s.
-int: Unknown. Can be 0, 1, 2 or 3.
+int: Flag. Can be 0, 1, 2 or 3.
+
+This last parameter seems to be a behaviour flag, after thorough testing I have come up with this:
+0 - Ped follows specified point route and stops
+1 - Ped follows specified point route, turns around after reaching end of point route and follows point route back to beginning (-> stops)
+2 - Ped patrols specified point route indefinitely (similar behavior to previous flag but in a loop)
+3 - Ped patrols specified point route indefinitely, but disregards point route when returning to first point (walks directly towards it)
+- Kolopsel
+
 Example:
```

Comment:
If possible, please convert this example into a Lua example as showcased here:
https://github.com/citizenfx/natives/blob/master/.github/CONTRIBUTING.md